packages/component-library/draft/Kaizen/Modal/presets/ConfirmationModal.scss

```diff
@@ -2,11 +2,6 @@
 @import "~@cultureamp/kaizen-component-library/styles/layout";
 @import "~@cultureamp/kaizen-component-library/styles/responsive";
 
-// override Murmur global styles :(
-.header h1 {
-  color: $white;
-}
-
```

Comment:
I removed this as it didn't seem to do anything. It was also interfering with my ability to change the color of the title specifically for the cautionary modal. Does anyone see a problem with removing this styling here?

Reply:
@sebpearce @lijuenc @ellisjas

Reply:
The comment says that it was an override for murmur. I think murmur has styles which apply globally to all `h1` elements and that interfered with this Kaizen component's styles.

Reply:
I would assume it does do something when you see it in Murmur rather than Storybook... @lijuenc is right, there are likely global `h1` styles in Murmur that will make it some heinous colour instead of white if you remove it.

Instead of removing it, you could try putting a class on the `h1` when it's cautionary and targeting that with `.header h1.h1NameGoesHere` maybe?

Reply:
Thanks @sebpearce! Just paired with @lijuenc to get it right. 🙏